setup.py

```diff
@@ -16,7 +16,7 @@ def package_data(pkg, root):
 
 setup(
     name='recommender-xblock',
-    version='0.1',
+    version='0.2',
```

Comment:
I guess I'm a little confused by the release tag naming convention versus the version specification in setup.py -- it would be easier to grok if these two values were aligned